translations/en/README.md

```diff
@@ -4,6 +4,37 @@ A career guide to Front End Developers
 
 ##  What every Software Developer needs to know
 
+You need to know: **nothing on this life is easy**, but if do you keep your humility and open mind, the path is far less thorny.
+
+You will not become the most incredible developer of the universe just reading this guide, but can go far if you follow the advice of good and experienced people (which may or may not have contributed here).
+
+No one is better than you. People just have more experience, more so because the *"be good"* depends on the point of view.
+
+You can't build the most amazing code, performative and scalable from day to night, but you can do it if you don't give up.
```

Comment:
Aqui podia ser *You can't build the most amazing code, `performatic` and scalable*